components/Badge/badge.css

```diff
@@ -0,0 +1,48 @@
+@import url(../../docs-css/reset.css);
+@import url(../common-css/drawer.css);
+@import url(../common-css/container.css);
+@import url(../../style.css);
+/* @import url(../Avatar/avatar.css); */
+
+.avatar{
+    width:3rem;
+    border-radius: 50%;
+    margin: 0.5rem;  
+}
+.badge{
+    display: flex;
+    position: relative;
+}
+.badge > section{
```

Comment:
Suggestion:-
You can add a separate class here instead of directly pointing out to the immediate child element